src/Configuration/PlatesFormatterConfiguration.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Equip\Configuration;
+
+use Auryn\Injector;
+use Equip\ContentNegotiation;
+use Equip\Formatter\PlatesFormatter;
+
+class PlatesFormatterConfiguration implements ConfigurationInterface
```

Comment:
Is it really necessary to have this configuration any more? Now that we're not using general-purpose responders, it doesn't seem very necessary to configure `ContentNegotiation` with only one formatter.